SuperBuild/External_ITK.cmake

```diff
@@ -146,6 +145,8 @@ if(NOT DEFINED ITK_DIR AND NOT Slicer_USE_SYSTEM_${proj})
       -DModule_ITKReview:BOOL=ON
       -DModule_MGHIO:BOOL=ON
       -DModule_ITKIOMINC:BOOL=ON
+      -DModule_MorphologicalContourInterpolation:Bool=ON
```

Comment:
Maybe `BOOL` uppercase to be consistent with the other specifications?

Reply:
Done